Documentation/troubleshooting.rst

```diff
@@ -166,12 +171,12 @@ minute.
 Monitoring Packet Drops
 -----------------------
 
-Cilium provides extensive visibility into 
-
-When connectivity is not as it should be. A main cause can be unwanted packet
-drops on the networking level. There can be various causes for this. The tool
+Sometimes you may experience broken connectivity, which may be due to a
+number of different causes. A main cause can be unwanted packet drops on
+the networking level. The tool
 ``cilium monitor`` allows you to quickly inspect and see if and where packet
-drops happen.
+drops happen. Following is an example output (use `kubectl` as in previous
```

Comment:
Could you switch this to say "use `kubectl exec`" so it's very clear which aspect of kubectl we want to use.
Similar to the other comment, it might also be worth explaining how to get the correct pod. Since we're providing instructions for both k8s and non-k8s, it might be easiest to use tabs with different snippets in each.

Reply:
Thanks for the review!

Changed `kubectl` to `kubectl exec`.